hydra-tui/src/Hydra/TUI/Handlers.hs

```diff
@@ -135,15 +136,35 @@ handleVtyEventsInitializingScreen cardanoClient hydraClient e = do
           id .= InitializingHome
         _ -> pure ()
       zoom commitMenuL $ handleFormEvent (VtyEvent e)
+    ConfirmingAbort i -> do
+      case e of
+        EvKey KEsc [] -> id .= InitializingHome
```

Comment:
`id .=` is always confusing me.. why not simply `put`?

Reply:
agree, but it is to keep it consistent with the rest.

do you want to include this refactor as part of this PR?